libraries/SPIFFS/src/SPIFFS.h

```diff
@@ -14,26 +14,37 @@
 #ifndef _SPIFFS_H_
 #define _SPIFFS_H_
 
+#pragma once
+
 #include "FS.h"
+#include "FSImpl.h"
```

Comment:
these two includes should not be in the header, but in the CPP file instead. Nothing in the header depends on them :)

Reply:
The header **does** depend on them, but they are also included via vfs_api.h, so at most they are redundant. For the same reason they don't need to be included in the .cpp file (because they are included via vfs_api.h which is included via SPIFFS.h).

Whatever, I have removed them from SPIFFS.h, but I haven't added them to SPIFFS.cpp.